cmd/commands.md

```diff
@@ -0,0 +1,3614 @@
+--aggregator-log-level string             log level to use with signature aggregator (default "Debug")
+--aggregator-log-to-stdout use            stdout for signature aggregator logs
+--balance uint                            set the AVAX balance of the validator that will be used for continuous fee on P-Chain
+--blockchain-genesis-key use              genesis allocated key to pay fees for completing the validator's registration (blockchain gas token)
+--blockchain-key string                   CLI stored key to use to pay fees for completing the validator's registration (blockchain gas token)
+--blockchain-private-key string           private key to use to pay fees for completing the validator's registration (blockchain gas token)
+--bls-proof-of-possession string          set the BLS proof of possession of the validator to add
+--bls-public-key string                   set the BLS public key of the validator to add
+--cluster string                          operate on the given cluster
+--create-local-validator create           additional local validator and add it to existing running local node
+--default-duration                        (for Subnets, not L1s) set duration so as to validate until primary validator ends its period
+--default-start-time                      (for Subnets, not L1s) use default start time for subnet validator (5 minutes later for fuji & mainnet, 30 seconds later for devnet)
+--default-validator-params                (for Subnets, not L1s) use default weight/start/duration params for subnet validator
+--delegation-fee uint16                   (PoS only) delegation fee (in bips) (default 100)
+--devnet operate                          on a devnet network
+--disable-owner string                    P-Chain address that will able to disable the validator with a P-Chain transaction
+--endpoint string                         use the given endpoint for network operations
+-e, --ewoq use                            ewoq key [fuji/devnet only]
+-f, --fuji testnet                        operate on fuji (alias to testnet
+-h, --help help                           for addValidator
+-k, --key string                          select the key to use [fuji/devnet only]
+-g, --ledger use                          ledger instead of key (always true on mainnet, defaults to false on fuji/devnet)
+--ledger-addrs strings                    use the given ledger addresses
+-l, --local operate                       on a local network
+-m, --mainnet operate                     on mainnet
+--node-endpoint string                    gather node id/bls from publicly available avalanchego apis on the given endpoint
+--node-id string                          node-id of the validator to add
+--output-tx-path string                   (for Subnets, not L1s) file path of the add validator tx
+--partial-sync set                        primary network partial sync for new validators (default true)
```

Comment:
Some of the flags are being generated without a type, causing the description to split (see original line 152)
```suggestion
-l, --local none                          operate on a local network
-m, --mainnet none                        operate on mainnet
--node-endpoint string                    gather node id/bls from publicly available avalanchego apis on the given endpoint
--node-id string                          node-id of the validator to add
--output-tx-path string                   (for Subnets, not L1s) file path of the add validator tx
--partial-sync none                       set primary network partial sync for new validators (default true)
```